packages/firebase_remote_config/example/lib/main.dart

```diff
@@ -0,0 +1,91 @@
+import 'dart:async';
+
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(new MaterialApp(
+      title: 'Remote Config Example',
+      home: new FutureBuilder<RemoteConfig>(
+        future: setupRemoteConfig(),
+        builder: (BuildContext context, AsyncSnapshot<RemoteConfig> snapshot) {
+          switch (snapshot.connectionState) {
```

Comment:
If you remote the `retrieveConfig` call from `setupRemoteConfig` this is going to be fast and can't fail, so I'm not sure the error handling is necessary.  It might be cleaner to just `return snapshot.hasData ? new WelcomeWidget(remoteConfig: snapshot.data) : new Container()`; and then you could get rid of the `ConnectionStateWidget`

Reply:
Done